tests/unit/nupic/algorithms/cla_classifier_test.py

```diff
@@ -28,7 +28,9 @@
 import unittest2 as unittest
 
 import numpy
+import tempfile
 
+from nupic.bindings.proto import ClaClassifier_capnp
```

Comment:
OH you have to do `import capnp` before importing any schema. When you do `import capnp` it sets up the import hooks so that Python knows how to find and parse the `.capnp` files.